ArduPlane/mode.h

```diff
@@ -745,6 +756,8 @@ class ModeTakeoff: public Mode
 
     bool does_auto_throttle() const override { return true; }
 
+    bool is_taking_off() const override { return true; }
```

Comment:
I think this is too simplistic, need to check if takeoff has completed and the vehicles is loitering. Check for `AP_FixedWing::FlightStage::TAKEOFF`.

Reply:
Flight_stage is "NORMAL" for all flight modes except TAKEOFF and AUTO.. unless VTOL is overriding it

Reply:
changed to using flight stage in mode.cpp